rest_framework_json_api/filters/queryvalidation.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from rest_framework.exceptions import ValidationError
+from rest_framework.filters import BaseFilterBackend
+
+
+class QueryValidationFilter(BaseFilterBackend):
```

Comment:
So it is not confused with query set validation I prefer we are a bit more explicit with the name.

e.g. QueryParameterValidationFilterBackend